database/migrations/20230824105507_create_keywords_table.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE "keywords" (
+     "id" uuid DEFAULT uuid_generate_v1(),
+     "user_id" uuid NOT NULL,
+     "keyword" text NOT NULL,
```

Comment:
Let's add an [INDEX](https://www.postgresql.org/docs/current/indexes-intro.html) on that column as it will be used for searching